### PR TITLE
DEVOPS-269: add make target to push all images and extend .travis.yml…

### LINKED ISSUE #27: Regularly rebuild all images
To get the latest bugfixes and security updates all images must be regularly rebuilt. Especially Wordpress which doesn't use a specific version but only the major version.

### PR DESCRIPTION
**Pull request by kr1sp1n**
… to build-all and push-all if CI cron job
Closes #27 and #46

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -32,3 +32,4 @@ script:
   # push docker images to registry if master branch:
   - make push -e CURRENT_BRANCH="$BRANCH" -e ONLY_BRANCH="master"
   - make push -e CURRENT_BRANCH="$BRANCH" -e ONLY_BRANCH="master" -e IMAGE_PREFIX="docker.io/avenga"
+  - if [ "$TRAVIS_EVENT_TYPE" == "cron" ]; then make build-all push-all -e CURRENT_BRANCH="$BRANCH" -e ONLY_BRANCH="master"; fi
```
Comment (bemeyert) [inline .travis.yml:35]: Build and push should go the avenga and 7val Docker-namespace.Also a test-all target is missing

File: .travis.yml [hunk 2]
```diff
@@ -32,3 +32,8 @@ script:
   # push docker images to registry if master branch:
   - make push -e CURRENT_BRANCH="$BRANCH" -e ONLY_BRANCH="master"
   - make push -e CURRENT_BRANCH="$BRANCH" -e ONLY_BRANCH="master" -e IMAGE_PREFIX="docker.io/avenga"
+  - > 
+    if [ "$TRAVIS_EVENT_TYPE" == "cron" ]; then 
+    make test-all build-all push-all -e CURRENT_BRANCH="$BRANCH" -e ONLY_BRANCH="master";
```
Comment (bemeyert) [inline .travis.yml:37]: The order is wrong: First build, then test and push at last.

Comment (bemeyert) [inline .travis.yml:37]: The parameter `-e IMAGE_PREFIX="docker.io/7val` should be added here. When we change all ocurrences from 7val to avenga the images would no longer be pushed to the 7val namespace.

Comment (kr1sp1n) [reply .travis.yml:37]: 🤦 Thanks!

Comment (kr1sp1n) [reply .travis.yml:37]: Yes, I had this already in mind. This will be added to #92.  Thanks for the hint.

**Pull request merged**